compiler/src/dmd/astbase.d

```diff
@@ -442,12 +442,19 @@ struct ASTBase
     extern (C++) final class StaticAssert : Dsymbol
     {
         Expression exp;
-        Expression msg;
+        Expressions* msg;
```

Comment:
Maybe call it `msgs`?

Reply:
If this requires many changes it could be left for a subsequent PR.

Reply:
@RazvanN7 now done and removed style changes. Also there is a spec PR.